vignettes/adrs_gcig.Rmd

```diff
@@ -0,0 +1,814 @@
+      "`MOUSEANT`", "Received mouse antibodies", "Y", "Indicates if a prohibited therapy was received", "Derivation of `ANL02FL`",
+      "`CA50RED`", ">50% reduction from baseline", "Y", "Indicates response, but does not distinguish between CR and PR", "Not used in further derivations",
+      "`CANORM2X`", "CA125 normal, lab increased >=2x ULRR", "Y", "Indicates PD category A or C", "Derivation of PD category (`MCRIT1`/`MCRIT1ML`/`MCRIT1MN`)",
+      "`CNOTNORM`", "CA125 not norm, lab increased >=2x nadir", "Y", "Indicates PD category B", "Derivation of PD category (`MCRIT1`/`MCRIT1ML`/`MCRIT1MN`)"
+)
+
+    knitr::kable(list_supp)
+# styler: on
+    ```
+
+The above `SUPPRS` records refer only to records with `RS.RSCAT = "CA125"`.
+The exception is `QNAM = "MOUSEANT"`, which appears for both `RS.RSCAT = "CA125"` and `RS.RSCAT = "RECIST 1.1 - CA125"` (for the same visits).
+
+If the data are collected by other ways and similar information from `SUPPRS`
+dataset are not available, they should be derived in advance from `LB` CA-125
+measurements records. Information on whether the patient has received mouse
+antibodies should also be taken into account.
+
+# Required Packages
+
+The examples of this vignette require the following packages.
+
+```{r, warning=FALSE, message=FALSE}
+library(admiral)
+library(admiralonco)
+library(pharmaversesdtm)
+library(pharmaverseadam)
+library(metatools)
+library(dplyr)
+library(tibble)
+```
+
+# Programming Workflow
+
+-   [Read in Data](#readdata)
+-   [Pre-processing of Input Records](#input)
+-   [Analysis Flag Derivation](#anlfl)
+-   [CA-125 Progression](#pdca)
+-   [CA-125 Progression Category](#pdcacat)
+-   [CA-125 Best Confirmed Overall Response](#cborca)
+-   [Combined Best Unconfirmed Overall Response](#combubor)
+-   [Confirmed RECIST 1.1 Overall Response](#covrr11)
+-   [Combined Best Confirmed Overall Response](#combcbor)
+-   [Derive Confirmed Response Parameter](#cborca125)
+
+
+## Read in Data {#readdata}
+
+To start, all data frames needed for the creation of `ADRS` should be read into the environment. This will be a company specific process. Some of the data frames needed are `ADSL` and `RS`.
+
+For example purpose, the SDTM and ADaM datasets (based on CDISC Pilot
+test data)---which are included in `{pharmaversesdtm}` and `{pharmaverseadam}`---are used.
+
+```{r message=FALSE}
+adsl <- pharmaverseadam::adsl
+# GCIG sdtm data
+devtools::install_git("https://github.com/pharmaverse/pharmaversesdtm/", ref = "123_rs_ca125")
+rs <- pharmaversesdtm::rs_onco_ca125
+supprs <- pharmaversesdtm::supprs_onco_ca125
+
+rs <- combine_supp(rs, supprs)
+rs <- convert_blanks_to_na(rs)
+```
+
+```{r, eval=TRUE, echo=FALSE}
+dataset_vignette(
+  rs,
+  display_vars = exprs(USUBJID, RSTESTCD, RSCAT, RSSTRESC, VISIT, CA125EFL, CA2XULRR, CA50RED, CANORM2X, CNOTNORM, MOUSEANT)
+)
+```
+
```

Comment:
I think we should restrict the records in `adsl` like in the ADRS vignette. Then we don't get so many records in `adrs` for subjects without RS data.

Reply:
updated